components/playlist/browser/resources/constants/style.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { css } from 'styled-components'
+
+export const playlistControlsAreaHeight = css`
```

Comment:
So I think this would be a bit more idiomatic if we defined the variable on one of the top level components (i.e. body, or a wrapper element in `App`). That way you wouldn't need to import this variable every time you need it.

Reply:
Yeah, that'd be much better. But we're using this value from two different frames, so I wasn't able to find out an efficient way to pass down the css variable to the subframe. Could you give some advice about this?

Reply:
Ah yeah, that definitely makes things more complicated :+1: Didn't realise this was for sharing variables between frames :smile: